app/initializers/stripe.js

```diff
@@ -1,14 +1,6 @@
-import config from '../config/environment';
-
-export function initialize() {
-  const application = arguments[1] || arguments[0];
-  const { stripe = {} } = config;
-
-  application.register('config:stripe', stripe, { instantiate: false });
```

Comment:
Will [this](https://github.com/smile-io/ember-cli-stripe/pull/194/files#diff-6339955f862dc343ebd4415d5549cde4d9ed2553b314297dcb9e7b5caba21799R67) still work if we remove this completely?
If we want to remove completely, we should remove the initializer completely and probably just read from `config/environment` in the service as at the top of this file.

Reply:
Done! I tested it and it worked!

Reply:
Can I ask how did you test this pls? 
I would assume that without this `stripeConfig` on the service will be `undefined` resulting in an error because the API key is missing.

Reply:
You are 100% right. I was just upgrading and downgrading plans, I need to do something like updating the credit card to really call the service. Thanks for pointing this out. I have restored the file, and it's now working.